zulipterminal/config/symbols.py

```diff
@@ -9,6 +9,9 @@
 
 INVALID_MARKER = "✗"  # BALLOT X, U+2717 (Dingbats)
 DIRECT_MESSAGE_MARKER = "§"  # SECTION SIGN, U+00A7 (Latin-1 supplement)
+ALL_MESSAGES_MARKER = "≡"
```

Comment:
Before merging, it would be good to have a similar comment for this symbol, ie. with the unicode name, hex, etc.

The other symbols you added aren't as important since they're currently standard letters/punctuation.

Reply:
I'll add a comment similar to the other ones.